### PR TITLE
Users/vivasa/publishsymbol/manifest

### PR DESCRIPTION
**Pull request by viralpandya**
**Task name**: PublishSymbolsV2

**Description**: Added manifest for publishsymbols

**Documentation changes required:** (Y/N) 

**Added unit tests:** (Y/N) 

**Attached related issue:** (Y/N) 

**Checklist**:
- [x] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [ ] Checked that applied changes work as expected

### CODE REVIEW

**Comment by viralpandya** [comment]
Verified manifest for the Publish Symbol task
image

**Comment by viralpandya** [comment]
Its verified and tested. Its working as expected

**Review by carl-tanner — APPROVED** [review]

File: Tasks/PublishSymbolsV2/PublishSymbols.ts [hunk 1]
```diff
@@ -37,7 +38,16 @@ export async function run(clientToolFilePath: string): Promise<void> {
...
         let personalAccessToken = tl.getVariable("ArtifactServices.Symbol.PAT");
-        const connectedServiceName = tl.getInput("ConnectedServiceName", false);        
+        const connectedServiceName = tl.getInput("ConnectedServiceName", false);
+        const manifest = tl.getInput("Manifest", false); 
+        if(manifest && !fileExists(manifest)) {
+            throw new Error(tl.loc("ManifestFileNotFound", manifest));
+        }
+        else
+        {
+            tl.debug("Manifest file found at: " + manifest);
```
Comment (quanterion) [inline Tasks/PublishSymbolsV2/PublishSymbols.ts:48]: looks like we will have debug message even if no manifest provided

Comment (viralpandya) [reply Tasks/PublishSymbolsV2/PublishSymbols.ts:48]: Thank you for correction. Here is the refactor PR https://github.com/microsoft/azure-pipelines-tasks/pull/20834. Please review and will be happy to know improvements as well.

**Pull request merged**